clusterloader2/pkg/chaos/nodes.go

```diff
@@ -121,7 +121,7 @@ func (k *NodeKiller) pickNodes() ([]v1.Node, error) {
 	for i := range prometheusPods {
 		if prometheusPods[i].Spec.NodeName != "" {
 			nodesHasPrometheusPod.Insert(prometheusPods[i].Spec.NodeName)
-			klog.Infof("%s: Node %s removed from killing. Runs pod %s", k, prometheusPods[i].Spec.NodeName, prometheusPods[i].Name)
+			klog.V(1).Infof("%s: Node %s removed from killing. Runs pod %s", k, prometheusPods[i].Spec.NodeName, prometheusPods[i].Name)
```

Comment:
While I understand the motivation of changing verbotisty of all logs (I didn't think about all changes in this PR), I have a concern that it won't persist for long time. i.e. my fear is that we will soon add a new `klog.Infof(...)` logs that will break your assumptions again.

So while I agree with switching some logs to V(2), I'm not sure that changing other to V(1) actually makes sense.
@mm4tt - thoughts?

Reply:
It would be helpful to have a guideline. The current levels were mainly set based on the importance and the amount of output.  My original motivation was to be able to disable the large amount of runtime measurement information generated in wait_for_pods.go, wait_for_pvs.go, wait_for_pvcs.go and wait_for_nodes, the detailed latency metrics (phase_latency.go:127) and the cluster nodes information (./util/cluster.go:70) and make them V(2).   I'll undo those V(1) changes and keep V(2).

Reply:
Removed the default level set and reverted all the V(1) changes.

Reply:
Yeah, switching some (the ones spamming the logs most) to V(2) make sense. 
We'll still run our CI/CD tests with full logs, as it's really helpful for debugging.  But I understand that other users may want to set verbosity to something higher.